src/borg/_hashindex.c

```diff
@@ -68,7 +68,7 @@ static int hash_sizes[] = {
 };
 
 #define HASH_MIN_LOAD .25
-#define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_LOAD 0.95  /* don't go higher than 0.75, otherwise performance severely suffers! */
```

Comment:
> If you change HASH_MAX_LOAD, do a full text search for it, there is another place depending on its value.

Had, a look. All I can see is the comment next to the value and `docs/internals.rst`.
I would update both once I identify a good value for this constant. Let me know if there's any places I've missed.